<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="i_reportDirL">
       <property name="text">
        <string>Директория сохранения отчёта:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLineEdit" name="reportDirLE"/>
       </item>
       <item>
        <widget class="QToolButton" name="reportDirTB">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="setLabelPrinterPB">
       <property name="text">
        <string>Выбрать принтер
для этикеток</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setReportPrinterPB">
       <property name="text">
        <string>Выбрать принтер
для отчётов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeNumPB">
       <property name="text">
        <string>Изменить номер
проверки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetSettPB">
       <property name="text">
        <string>Сбросить все
 настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delUsersPB">
       <property name="text">
        <string>Сбросить всех
пользователей</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeAdminPassPB">
       <property name="text">
        <string>Сменить пароль
администратора</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="newUsersCB">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Разрешить создание
новых пользователей</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="i_prodNameL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Жгут:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prodNameCB"/>
     </item>
     <item>
      <widget class="QToolButton" name="addProdTB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icon/plus.png</normaloff>:/icon/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delProdTB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icon/minus.png</normaloff>:/icon/minus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="i_labelPrinterL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Принтер этикеток: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPrinterL">
       <property name="text">
        <string> - </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="i_reportPrinterL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Принтер отчёта:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reportPrinterL">
       <property name="text">
        <string> - </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="i_circuitL">
         <property name="text">
          <string>Цепи:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addTB">
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icon/plus.png</normaloff>:/icon/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="delTB">
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icon/minus.png</normaloff>:/icon/minus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="editTB">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/icon/edit.png</normaloff>:/icon/edit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="circuitTV"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
